modules/misc/qt.nix

```diff
@@ -141,6 +153,14 @@ in {
     qt.style.package = mkIf (cfg.style.name != null)
       (mkDefault (stylePackages.${toLower cfg.style.name} or null));
 
+    qt.pluginPath =
+      builtins.map (prefix: "${config.home.profileDirectoty}/${prefix}") [
+        pkgs.qt6.qtbase.qtPluginPrefix
+        pkgs.qt6.qtbase.qtQmlPrefix
+        pkgs.qt5.qtbase.qtPluginPrefix
+        pkgs.qt5.qtbase.qtQmlPrefix
+      ];
```

Comment:
`pkgs.qt*.qtbase.qtQmlPrefix` goes in `QML2_IMPORT_PATH`, it needs to be a separate list.

Reply:
Oh sorry